models/_generate_layers.py

```diff
@@ -81,7 +81,7 @@ def setup():
         if param.is_module_init_arg():
           print(param.get_module_param_docstring(indent="    "), file=f)
       print('    """', file=f)
-      print(f"    super().__init__({'**kwargs' if issubclass(cls_base, LayerBase) else ''})", file=f)
+      print(f"    super().__init__({sig.get_super_call_code_str()})", file=f)
```

Comment:
```suggestion
      print(f"    {sig.get_super_call_code_str()}", file=f)
```
(This is just to have it simple and clean as I suggested before. Also to have it consistent to other code, see other functions `..._code_str()`.)